common/spec/dependabot/config/update_config_spec.rb

```diff
@@ -33,6 +33,17 @@
         expect(ignored_versions).to eq([">= 14.14.x, < 15"])
       end
     end
+
+    context "with a wildcard dependency name" do
```

Comment:
Idea: I'd dig a test where the target dependency matches multiple conditions.
I think the expectation is we'll `flat_map` them out, and I think it will work... it just wasn't possible before, but is now.

The behaviour of multiple matches gets more interesting in https://github.com/dependabot/dependabot-core/pull/3513 , but that branch doesn't have wildcards yet! 🐔🥚

Reply:
Added 👍